### PR TITLE
Avoid errors when the process does not have enough permissions to change attributes

### PR DESCRIPTION
**Pull request by hhorak**
I tried to rebase the original #120 by an innovative way, but it ended bad. This PR is the same as #120.

### CODE REVIEW

**Comment by omron93** [comment]
[test]

**Comment by pkubatrh** [comment]
[test]

File: bin/fix-permissions [hunk 1]
```diff
@@ -3,11 +3,22 @@
...
+
+# If argument does not exist, script will still exit with 0,
+# but at least we'll see something went wrong in the log
+if ! [ -e "$1" ] ; then
+  echo "ERROR: File or directory $1 does not exist." >&2
+  # We still want to end successfully
+  exit 0
+fi
+
+# Using --quiet to avoid error messages when there are not enough permissions
```
Comment (pkubatrh) [inline bin/fix-permissions]: Can this comment be removed? Since we are not using `--quiet` anymore...

**Comment by pkubatrh** [comment]
>Can this comment be removed? Since we are not using --quiet anymore...

Since it is just a comment I have removed if for you and pushed the changes manually.

**Pull request closed without merging**